### PR TITLE
Update main.js

### PR DESCRIPTION
**Pull request by oliveira-emmanuel**
Remove Comments

### CODE REVIEW

File: src/scripts/main.js [hunk 1]
```diff
@@ -16,7 +16,7 @@ document.addEventListener("DOMContentLoaded", function () {
   })
 
 
-  //Seção de Atrações programação das Abas
```
Comment (oliveira-emmanuel) [inline src/scripts/main.js:19]: Removi os comentários

**Review by oliveira-emmanuel — COMMENTED** [review]
good

File: src/scripts/main.js (hunk 1 above)
Comment (emmanuelmarcosdeoliveira) [reply src/scripts/main.js:19]: ok

Comment (emmanuelmarcosdeoliveira) [reply src/scripts/main.js:19]: ok

**Review by emmanuelmarcosdeoliveira — APPROVED** [review]

**Pull request merged**